app/views/contexts/summary/block.py

```diff
@@ -12,15 +13,18 @@ def __init__(
         list_store,
         metadata,
         response_metadata,
-        schema,
+        schema: QuestionnaireSchema,
```

Comment:
Any reason for type hinting for just schema? I was thinking none or all

Reply:
was just using it for getting hold of method names from `schema`, will remove